.github/workflows/conformance.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: run end-to-end tests that use the `dns-test` framework
         run: just e2e-tests
 
+      - name: check public tests that use the `dns-test` framework
+        run: just public-dns-tests-check
```

Comment:
Note that this still seems to be using the old name of the `just` target.

Reply:
thanks for catching that